examples/template.yaml

```
@@ -4,12 +4,27 @@ version: 0.1.0
\ No newline at end of file
+    - $ {{GLOBAL_VAR}} f{local_arg} --f{local_arg_2}
+    config:
+      deprecated: True
+      context_settings:
+        help_option_names: ["-h", "--helpme"]
+  
+  debug:
+    help: ""
+    template: common_args
+    run: |
+      print(f"arg1: {local_arg}, arg2: --{local_arg_2}")
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)